pkg/generator/certificate.go

```diff
@@ -1,32 +1,44 @@
 package generator
 
 import (
-	"bytes"
 	"crypto/ecdsa"
 	"crypto/elliptic"
 	"crypto/rand"
 	"crypto/x509"
 	"crypto/x509/pkix"
 	"encoding/pem"
 	"math/big"
+	"net"
 	"time"
-
-	"github.com/ForgeRock/secret-agent/api/v1alpha1"
 )
 
-func generateCA(alias *v1alpha1.AliasConfig) ([]byte, error) {
+//RootCA data trasfer object for root CAs
```

Comment:
@snarlysodboxer LMK what you think. 
Note the new struct RootCA. This is needed to issue signed certs with the rootCA

Reply:
see controllers/webhook_cabundler.go

Reply:
Good idea on the new type! I hadn't thought of doing it that way.